build-customization.rst

```diff
@@ -0,0 +1,72 @@
+.. post:: Nov 10, 2022
+   :tags: announcement, feature, builds
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      We released user-defined commands (``build.commands`` config key)
+      which allow users to overwrite the build process completely.
```

Comment:
```suggestion
      We released a user-defined command (``build.commands`` config key)
      which allow you to overwrite the build process completely.
```

Reply:
Hrm, it's not "a user-defined command", but I think it not correct what I wrote either.

Probably,

```suggestion
      We released a new config key (``build.commands``) to specify user-defined commands
      which allow users to overwrite the build process completely.
```

is that better?

Reply:
Yes, haha, that's so much better, I completely forget the meaning of the sentence, just wanted it to sound good :see_no_evil:

Reply:
I edited your comment's suggestion to fix a typo, pretty crazy workflow :scream: